implementation/src/main/java/io/smallrye/mutiny/helpers/queues/Queues.java

```diff
@@ -45,28 +60,23 @@ public static <T> Supplier<Queue<T>> getXsQueueSupplier() {
      * @return the supplier.
      */
     public static <T> Supplier<Queue<T>> get(int bufferSize) {
```

Comment:
capacity

Reply:
I'm harmonizing to `capacity` / `chunkSize` as in JCTools signatures